pom.xml

```diff
@@ -19,7 +19,7 @@
     <groupId>com.amazonaws</groupId>
     <artifactId>amazon-neptune-sigv4-signer</artifactId>
     <packaging>jar</packaging>
-    <version>3.0.1-SNAPSHOT</version>
+    <version>3.0.2</version>
```

Comment:
we never released 3.0.1. you can go to 3.0.1, release all this to maven central and afterwards, bump to 3.0.2-SNAPSHOT for a future version

Reply:
Updated the version to 3.0.1